File: java/src/main/java/co.edu.sena.capitulo90/src/main/java/co/edu/sena/capitulo9/Ejercicio01.java
package co.edu.sena.capitulo9;

import java.util.Scanner;

public class Ejercicio01 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		double lado;
		System.out.print("lado: ");
		lado = teclado.nextFloat();
		cuadrado(lado);

	}

	private static void cuadrado(double lado) {
		double area;
		double perimetro;
		area = lado * lado;
		perimetro = lado * 4;
		System.out.println("Area: " + area);
		System.out.println("Perimetro" + perimetro);

	}

}

File: typescript/src/ejercicios/ejercicio01.ts
(function () {
    let area = 0;
    let perimetro = 0;
    let lado;
    lado = parseInt(prompt("lado: "));

    cuadrado(lado);

    function cuadrado(lado: number) {
    let l;
    let area;
    perimetro = 4 * lado;
    area = lado * lado;
    perimetro = lado * 4;
    lado += 1;
    console.log("Area: " + area);
    console.log("Perimetro" + perimetro);
   }
    
})();
